test/integration/controlplane/transformation/transformation_testcase.go

```diff
@@ -167,6 +201,7 @@ func (e *transformTest) getRawSecretFromETCD() ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to read %s from etcd: %v", secretETCDPath, err)
 	}
+	e.logger.Logf("path: %s", secretETCDPath)
```

Comment:
I would rename the functions and variables to be more generic:

```suggestion
	e.logger.Logf("path: %s", etcdPath)
```